releases/latest/postgresql-k8s-bundle.yaml

```diff
@@ -3,31 +3,61 @@ applications:
     channel: 1/edge
     charm: pgbouncer-k8s
     constraints: arch=amd64
-    revision: 70
+    revision: 146
     scale: 1
+    trust: true
   postgresql-k8s:
     channel: 14/edge
     charm: postgresql-k8s
     constraints: arch=amd64
-    revision: 136
+    revision: 250
     scale: 2
     trust: true
-  tls-certificates-operator:
-    channel: latest/stable
-    charm: tls-certificates-operator
+  self-signed-certificates:
+    channel: latest/edge
+    charm: self-signed-certificates
     constraints: arch=amd64
     num_units: 1
     options:
       ca-common-name: postgresql_k8s_bundle_ca
-      generate-self-signed-certificates: true
-    revision: 22
+    revision: 140
+  data-integrator:
+    channel: latest/edge
+    charm: data-integrator
+    constraints: arch=amd64
+    revision: 26
+    scale: 1
+  s3-integrator:
+    channel: latest/edge
+    charm: s3-integrator
+    constraints: arch=amd64
+    revision: 17
+    scale: 1
+  grafana-agent-k8s:
```

Comment:
JFMI, can we use the same grafana-agent-k8s for both pb and pgb on K8s?

Reply:
I think so, but I haven't double checked.